ui/templates/background-images.css

```diff
@@ -20,21 +20,16 @@
 .sk-search-box form .sk-search-box__icon:before {
   background-image: url({% static "images/search-icon.svg" %});
 }
-
-.certificate .certificate-body.verified {
-  background-image: url({% static "images/mm_watermark2.png" %});
-
+.b2b {
+  background-image: url({% static "images/b2b_section.png" %});
 }
-
-@media print {
-  .certificate .certificate-body.verified {
-    -webkit-print-color-adjust: exact !important;   /* Chrome, Safari */
-    color-adjust: exact !important;  /* Firefox */
-    background-image: url({% static "images/mm_watermark2.png" %}) !important;
-    background-size: cover !important;
+@media (min-width: 1200px) {
+  .certificate .accomplishment-rendering {
+    background-image: url({% static "images/micromasters_certificate_watermark.svg" %}) !important;
   }
 }
-
-.b2b {
-  background-image: url({% static "images/b2b_section.png" %});
+@media print {
```

Comment:
@HamzaIbnFarooq these two lines are needed here, could you add that back?
These are required to show the background watermark in the print background.

```
-webkit-print-color-adjust: exact !important;   /* Chrome, Safari */	
color-adjust: exact !important;  /* Firefox */```